src/state_point.F90

```diff
@@ -461,6 +694,11 @@ subroutine write_source_point()
     integer(HID_T) :: file_id
     character(MAX_FILE_LEN) :: filename
 
+    if (dd_run) then
+      if (master) call warning('Source bank writing not implemented for DD runs.')
```

Comment:
I'm a little confused.  This warning suggest that writing the source is not implemented but then why is there all this DD specific code below?